projects/packages/forms/src/blocks/contact-form/components/jetpack-manage-responses-settings.js

```diff
@@ -12,7 +12,11 @@ const JetpackManageResponsesSettings = () => {
 			<InspectorHint>
 				{ __( 'Manage and export your form responses in WPAdmin:', 'jetpack-forms' ) }
 			</InspectorHint>
-			<Button variant="secondary" href={ RESPONSES_PATH } target="_blank">
+			<Button
+				variant="secondary"
+				onClick={ () => window.open( RESPONSES_PATH, '_blank' ) }
```

Comment:
Changed from href to onClick to force this to render as a button so styling is consistent with other buttons. Otherwise, this component with href renders as a link with different styling.